fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java

```diff
@@ -178,6 +178,11 @@ protected AnalysisInfo getReAnalyzeRequiredPart(AnalysisInfo jobInfo) {
         AnalysisManager analysisManager = Env.getServingEnv().getAnalysisManager();
         TableStatsMeta tblStats = analysisManager.findTableStatsStatus(table.getId());
 
+        // Skip tables that are too width.
+        if (table.getColumns().size() > StatisticsUtil.getAutoAnalyzeTableWidthThreshold()) {
```

Comment:
do we need add ut to check the Threshold work well?

Reply:
Added